src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -576,6 +576,22 @@
     )
     private Set<ClaimsCostFrom> divorceClaimFrom;
 
+    @CCD(
+        label = "Uploaded supporting documents",
+        hint = "Such as marriage/civil union certificate, proof of name changes, certified translations etc.",
+        typeOverride = Collection,
```

Comment:
why are the overrides needed?

Reply:
I think for collection it is required for ccd to know the type